src/ensembl/src/analyticsHelper.ts

```diff
@@ -0,0 +1,14 @@
+export enum AnalyticsCategory {
+  GLOBAL = 'Global'
+}
+
+export const buildAnalyticsObject = (category: string) => (data: {}) => ({
```

Comment:
`data: {[key: string]: any}` ?

Reply:
I have changed `data` to use an analytics object type.